src/haskell/devcontainer-feature.json

```
@@ -0,0 +1,54 @@
+{
+    "id": "haskell",
+    "version": "0.0.1",
```

```suggestion
    "version": "1.0.0",
```

We have been publishing all the Features in this repo from major version `v1`. Can we keep this in sync? Thanks!

---

done.